### PR TITLE
DOC-3710-1: Quick links to 4.x docs from landing page

### PR DESCRIPTION
**Pull request by AaronForinton**
The Corda 4 box now links directly to latest versions of OS, Enterprise, and CENM. Will need to be updated with releases.

### CODE REVIEW

File: themes/doks/layouts/index.html [hunk 1]
```diff
@@ -163,10 +163,12 @@ <h3 class="card-title">Corda 5 Developer Preview</h3>
         <div class="card no-border h-100">
           <div class="card-body">
             <h3 class="card-title">Corda 4</h3>
-            <p>Everything you need to know about the Corda 4 platform series - read the comprehensive documentation sets for Corda open source 4.x, Corda Enterprise 4.x, and Corda Enterprise Network Manager 1.x.</p>
-          </div>
-          <div class="card-footer">
-            <a href="en/platform/corda/4.8/open-source.html" class="btn rounded">Ready, set, read!</a>
+            <p>Everything you need to know about the Corda 4 platform series - read the comprehensive documentation sets for:</p>
+            <ul>
+              <li style="text-align: left;"><a href="en/platform/corda/4.8/open-source.html">Corda OS 4.8</a></li>
```
Comment (michelemarsden) [inline themes/doks/layouts/index.html:168]: Should ENT go above OS?

Comment (AaronForinton) [reply themes/doks/layouts/index.html:168]: Good point!

File: themes/doks/layouts/index.html [hunk 2]
```diff
@@ -175,7 +177,7 @@ <h3 class="card-title">Corda 4</h3>
           <div class="card-body">
             <h3 class="card-title">API reference</h3>
             <p>Find complete documentation reference in Javadoc and KDoc format for all publicly exposed APIs in the Corda 5 Developer Preview and the Corda 4
-             series (Corda open source 4.x, Corda Enterprise 4.x, and Corda Enterprise Network Manager 1.x).</p>
+              series (Corda open source 4.x, Corda Enterprise 4.x, and Corda Enterprise Network Manager 1.x).</p>
```
Comment (AaronForinton) [inline themes/doks/layouts/index.html:180]:
```suggestion
             series (Corda open source 4.x, Corda Enterprise 4.x, and Corda Enterprise Network Manager 1.x).</p>
```

**Review by michelemarsden — APPROVED** [review]
LGTM

**Pull request merged**